### ISSUE TITLE
Annoying error for missing CHANGELOG.md

### ISSUE THREAD

**Issue by frennkie** [janoside/btc-rpc-explorer#175]
**Describe the bug**

Error message regarding **CHANGELOG.md** not found being writen to logs.

```
btcexp:error Error 2379gsd7sgd334: Error: ENOENT: no such file or directory, open 'CHANGELOG.md', json: {"errno":-2,"code":"ENOENT","syscall":"open","path":"CHANGELOG.md"}
```

**Environment (please complete the following information):**

* btc-rpc-explorer@1.1.3
* node-v10.16.0-linux-armv7l
 
**Configuration file content**

```
cat .config/btc-rpc-explorer.env
# Host/Port to bind to
# Defaults: shown
BTCEXP_HOST=0.0.0.0
#BTCEXP_PORT=3002
# Bitcoin RPC Credentials (URI -OR- HOST/PORT/USER/PASS)
# Defaults:
#   - [host/port]: 127.0.0.1:8332
#   - [username/password]: none
#   - cookie: '~/.bitcoin/.cookie'
#   - timeout: 5000 (ms)
BTCEXP_BITCOIND_URI=bitcoin://raspibolt:<redacted>@127.0.0.1:8332?timeout=10000
#BTCEXP_BITCOIND_HOST=127.0.0.1
#BTCEXP_BITCOIND_PORT=8332
BTCEXP_BITCOIND_USER=raspibolt
BTCEXP_BITCOIND_PASS=<redacted>
#BTCEXP_BITCOIND_COOKIE=/path/to/bitcoind/.cookie
BTCEXP_BITCOIND_RPC_TIMEOUT=5000
# Password protection for site via basic auth (enter any username, only the password is checked)
# Default: none
BTCEXP_BASIC_AUTH_PASSWORD=<redacted>
# Select optional "address API" to display address tx lists and balances
# Options: electrumx, blockchain.com, blockchair.com, blockcypher.com
# If electrumx set, the BTCEXP_ELECTRUMX_SERVERS variable must also be
# set.
# Default: none
BTCEXP_ADDRESS_API=electrumx
BTCEXP_ELECTRUMX_SERVERS=tcp://127.0.0.1:50001
```

**To Reproduce**

Run `/usr/local/lib/nodejs/node-v10.16.0-linux-armv7l/bin/btc-rpc-explorer`

**Screenshots or Log Output**

```Loading env file: /home/bitcoin/.config/btc-rpc-explorer.env
  btcexp:app Starting BTC RPC Explorer, v1.1.3 +0ms
  btcexp:app Connecting to RPC node at 127.0.0.1:8332 +2ms
  btcexp:error Error 2379gsd7sgd334: Error: ENOENT: no such file or directory, open 'CHANGELOG.md', json: {"errno":-2,"code":"ENOENT","syscall":"open","path":"CHANGELOG.md"} +0ms
  btcexp:error Stack: Error: ENOENT: no such file or directory, open 'CHANGELOG.md' +0ms
  btcexp:app Connected via RPC to node. Basic info: version=190001, subversion=/Satoshi:0.19.0.1/, protocolversion=70015, services=0000000000000409 +240ms
```

**Comment by shawnyeager** [comment]
I'm not entirely sure that you're referring to the "(changelog)" link at the bottom. I'm seeing this when I click it:

```
marked(): input parameter is undefined or null

Error: marked(): input parameter is undefined or null
    at marked (/usr/lib/node_modules/btc-rpc-explorer/node_modules/marked/lib/marked.js:1552:11)
    at /usr/lib/node_modules/btc-rpc-explorer/routes/baseActionsRouter.js:1200:29
    at Layer.handle [as handle_request] (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/layer.js:95:5)
    at next (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/layer.js:95:5)
    at /usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:335:12)
    at next (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:275:10)
    at Function.handle (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:174:3)
    at router (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:47:12)
    at Layer.handle [as handle_request] (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:317:13)
    at /usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:335:12)
    at next (/usr/lib/node_modules/btc-rpc-explorer/node_modules/express/lib/router/index.js:275:10)
```

<img width="1198" alt="Screen Shot 2020-01-07 at 11 29 45 AM" src="https://user-images.githubusercontent.com/980297/71915276-041f0b00-3141-11ea-83aa-24e095da440a.png">

**Comment by frennkie** [comment]
It get the error when starting it.

Using the link at the bottom I also get the same as you.

**Comment by janoside** [comment]
After a couple of failed fix attempts this should be fixed in published v1.1.8. Both issues (the log output at startup and the page error when navigating to `/changelog`) should be addressed. Let me know if you see otherwise. Thanks for reporting this.
